packages/IMAPClient-Tests.package/ICEndPointCommandTest.class/instance/testRetrieveFlagsOnly.st

```diff
@@ -35,9 +34,13 @@ testRetrieveFlagsOnly
 	self assert: mailB flagged equals: true.
 	
 	"Asserts that the flags - in the maildir storage file - were also updated"
-	readWriter := ICFileReadWriter new.
-	mailA := readWriter get: '3' in: path.
-	mailB := readWriter get: '4' in: path.
+	
+	streamA :=	FileStream readOnlyFileNamed: path , FileDirectory slash, '3'.
+	mailA := ICEmail newFromJsonStream: streamA.
+	streamB := FileStream readOnlyFileNamed: path , FileDirectory slash, '4'.
+	mailB := ICEmail newFromJsonStream: streamB.
+	streamA close.
+	streamB close.
 	
 	self assert: mailA seen equals: false.
```

Comment:
Having one assertion per test is the better style.
Try to combine both conditions.

Reply:
We know this. This just conforms the test to the new interface, but I will change this to look better.